engine/lite/kafka/integration-test/src/it/scala/pl/touk/nussknacker/engine/lite/kafka/NuReqRespRuntimeBinTest.scala

```diff
@@ -1,50 +1,18 @@
 package pl.touk.nussknacker.engine.lite.kafka
 
 import com.typesafe.scalalogging.LazyLogging
-import io.circe.syntax._
-import org.apache.commons.io.FileUtils
 import org.scalatest.FunSuite
-import pl.touk.nussknacker.engine.build.ScenarioBuilder
-import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
-import pl.touk.nussknacker.engine.spel.Implicits._
+import pl.touk.nussknacker.engine.lite.kafka.NuRuntimeTestUtils.{saveScenarioToTmp, testCaseId}
+import pl.touk.nussknacker.engine.lite.kafka.sample.NuReqRespTestSamples.{jsonPingMessage, jsonPongMessage, pingPongScenario}
 import pl.touk.nussknacker.test.AvailablePortFinder
 import sttp.client.{HttpURLConnectionBackend, Identity, NothingT, SttpBackend, UriContext, basicRequest}
 
-import java.io.File
-import java.nio.charset.StandardCharsets
-
 class NuReqRespRuntimeBinTest extends FunSuite with BaseNuRuntimeBinTestMixin with LazyLogging {
```

Comment:
can we write some comment somewhere, what sbt task one should execute to get built required artifacts to run this test?

Reply:
this:
```
    IntegrationTest / Keys.test := (IntegrationTest / Keys.test).dependsOn(
      liteEngineKafkaRuntime / Universal / stage,
      liteEngineKafkaRuntime / Docker / publishLocal
    ).value,
```
is not a good comment? :)

Reply:
`stage` is enigmatic? I'll add this